nmrfx-core/src/main/java/org/nmrfx/math/VecBase.java

```diff
@@ -1036,7 +1036,7 @@ public byte[] getBytes() {
                     buffer[j++] = (byte) (intVal & 0xFF);
                 }
             }
-        } else if (useApache) {
+        } else if (useApache && cvec != null) {
```

Comment:
A lot of real Vectors had useApache set to true, but cvec was null, rvec was used since it was a real vector. Instead of changing useApache to false for real vectors because that could possible affect a lot of areas in the code, just added a null check here since this is the first time getBytes is actually being used, it may not have been tested before. Double check if this is okay

Reply:
I think the problem is that the whole  } else if (useApache) {  branch shouldn't be there.  If the Vec is not complex, we should not check useApache, and should always use data in rvec.